disttask/framework/dispatcher/dispatcher.go

```diff
@@ -460,6 +466,19 @@ func GenerateSchedulerNodes(ctx context.Context) ([]*infosync.ServerInfo, error)
 	return serverNodes, nil
 }
 
+func GenerateSchedulerNodes4Test() ([]*infosync.ServerInfo, error) {
```

Comment:
Could we put it to `framework_test.go`? It seems that only test use this function.